components/user-mgt/org.wso2.carbon.identity.user.profile/src/main/java/org/wso2/carbon/identity/user/profile/mgt/UserProfileAdmin.java

```diff
@@ -568,10 +568,9 @@ public void associateID(String idpID, String associatedID) throws UserProfileExc
         PreparedStatement prepStmt = null;
         String sql = null;
         int tenantID = CarbonContext.getThreadLocalCarbonContext().getTenantId();
-        String tenantAwareUsername = MultitenantUtils.getTenantAwareUsername(CarbonContext.getThreadLocalCarbonContext()
-                                                                          .getUsername());
-        String domainName = getDomainName(tenantAwareUsername);
-        tenantAwareUsername = getUsernameWithoutDomain(tenantAwareUsername);
+        String tenantAwareUsername = CarbonContext.getThreadLocalCarbonContext().getUsername();
+        String userStoreDomainName = getDomainName(tenantAwareUsername);
```

Comment:
Can't we get rid of private getDomainName() method and use IdentityUtil.extractDomainFromName() method?